middleware/index.ts

```diff
@@ -69,6 +67,9 @@ async function invokeAsyncFn(logger: Logger, cloudEvent: CloudEvent, headers: an
         isHttps = headers[X_FORWARDED_PROTO].toLowerCase() === 'https';
     }
 
+    // length changes after body content updates
+    delete headers['content-length'];
```

Comment:
Is there anything else we should consider deleting?

Reply:
Potentially - depends on what other clients send.  We know this header /definitely/ caused issues with re-invoking.